rector.php

```diff
@@ -20,18 +18,8 @@
     $rectorConfig->sets([LevelSetList::UP_TO_PHP_74]);
 
     $rectorConfig->skip([
-        // Falsely detect unassigned variables in code paths stopped by PHPUnit\Framework\Assert::markTestSkipped()
-        AddDefaultValueForUndefinedVariableRector::class => [
-            __DIR__ . '/tests/',
-        ],
-        // @see https://github.com/phpstan/phpstan-src/pull/2429
-        RemoveExtraParametersRector::class => [
-            __DIR__ . '/src/Operation/',
-        ],
-        // Assigns wrong type due to outdated PHPStan stubs
-        TypedPropertyFromAssignsRector::class => [
-            __DIR__ . '/src/Model/BSONIterator.php',
-        ],
+        // Do not use ternaries extensively
+        IfIssetToCoalescingRector::class,
         // Not necessary in documentation examples
         JsonThrowOnErrorRector::class => [
```

Comment:
False CS error: "Key specified for array entry; first entry has no key".
We can disable this rule for this file.

Reply:
Working on this now.

Reply:
Done ✅